packages/core/admin/ee/server/services/user.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const { pipe, castArray, map, toNumber, omit, pick, has } = require('lodash/fp');
+const _ = require('lodash');
```

Comment:
has, pick, and omit can be required from 'lodash/fp'

Reply:
same as above